src/core/plugins/oas31/oas3-extensions/fn.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @prettier
+ */
+import { immutableToJS } from "core/utils"
+
+export const isFileUploadIntended = (schema, mediaType = null) => {
+  if (
```

Comment:
Let's extract this into a config with the default of what we currently have in this code.